lib/game.rb

```diff
@@ -31,5 +32,29 @@ def start_game
       puts "Invalid option, enter p to play. Enter q to quit."
     end
   end
-end
 
+  def start_playing
+    
+    puts "game in session"
+
+  end
+
+  def ship_coordinates
+    while true
+      puts 'Enter coloumn letter (between A and D):'
+      placement_coloumn = gets.chomp.to_s.upcase
+      break if placement_coloumn.between?(0, 4)
```

Comment:
**Main Issue**
`while true` is bad practice and leaves room for infinite loops.
Maybe try something different like  `do while` block, do while loop is similar to while loop with the only difference that it checks the condition after executing the statements
```
loop do

 # code to be executed

break if Boolean_Expression

end
``` 

**Another (minor) Issue**
Line #46 spelling mistake `placement_coloumn`
Line #46 placement column -- while loop will never break because user is asked to enter a letter from A-D 
not a number from 0-4

Reply:
Thank you for catching these mistakes. Made some updates.